Say-Their-Name/say-their-names-web
# Fix profile back button

#### What does this PR do?
1) Passes the current pagination state to a profile page when clicked on from the profile list
2) Restores that state upon clicking `< BACK TO PROFILES`

#### Description of Task to be completed?
- FIxes the bug reported in issue  #67 

#### How should this be manually tested?
- Navigate to the profile list, select a profile on a given page: `n`
- Upon clicking `< BACK TO PROFILES` the pagination controls state should be restored and be on page `n` again.

#### What are the relevant Trello Card stories?
- https://trello.com/c/WcQyN3U1/106-fix-profile-back-button-behavior

#### Pull Request Type (check one)

#### Developer Checklist


#### Screenshots (if appropriate)
